dev-config.yaml

```diff
@@ -6,6 +6,10 @@
 #       all values with localhost:5000 below are set explicitly from the test
 #       job to an IP that is reachable instead.
 #
+# Disable network policy so that the test job is able to reach a local docker
+# registry
+networkpolicy:
```

Comment:
z2jh uses networkPolicy (with capital P) so let's use that: https://z2jh.jupyter.org/en/stable/resources/reference.html#hub-networkpolicy